draft-ietf-cbor-packed.md

```diff
@@ -376,16 +380,27 @@ Table setup can happen in one of two ways:
   case.
 
 * By one or more tags enclosing the packed content.
-  These can be defined to add to the packing tables that already apply
-  to the tag.  Usually, the semantics of the tag will be to prepend
-  items to one of the tables.
+  Each tag is usually defined to add to the packing tables that
+  already apply to the tag and apply the resulting augmented table when
+  unpacking the tag content.
```

Comment:
If I understand it right, this is equivalent to the slightly more verbose:

"Each tag is usually defined to add to the packing tables that already apply to the tag, and to apply the resulting augmented table when unpacking the tag content."

Correct?

Reply:
Thanks for the suggestion -- I made it even more explicit now.